Cue/app/actions/library.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+import type { PromiseAction } from './types';
+import LibraryApi from '../api/Library';
+
+async function loadLibrary(state): PromiseAction {
+  let library = eval(await LibraryApi.fetchLibrary(state));
```

Comment:
OK. If you were testing on `projectorange.club` it might have been because the DB stored procedures weren't updated, so the server was returning bad results, according to @ClaytonPassmore.